packages/relay/tests/lib/eth.spec.ts

```diff
@@ -3136,6 +3138,19 @@ describe('Eth calls using MirrorNode', async function () {
     expect(result).to.equal(numberTo0x(constants.TX_DEFAULT_GAS_DEFAULT));
   });
 
+  it('should perform estimateGas precheck', async function () {
```

Comment:
Add a check for null and undefined on `value` and `gasPrice` so we have coverage to confirm there's no errors.

Reply:
Added, but we have validations way before we even get in the method, so we're safe. If we try to pass something different from hex for `value` and `gasPrice`, relay will return validation error.